manual_jb/content/introduction.md

```diff
@@ -39,6 +39,7 @@ Below you can find three Jupyter notebook [research examples](notebooks.md) that
 2. [Joplin testbed](notebooks/Joplin_testbed/Joplin_testbed.md) ([zip](https://github.com/IN-CORE/incore-docs/blob/master/notebooks/Joplin_testbed.zip) file).
 3. [Lumberton testbed](notebooks/Lumberton_testbed/Lumberton_testbed.md) ([zip](https://github.com/IN-CORE/incore-docs/blob/master/notebooks/Lumberton_testbed.zip) file).
 4. [Seaside testbed](notebooks/Seaside_testbed/Seaside_testbed.md) ([zip](https://github.com/IN-CORE/incore-docs/blob/master/notebooks/Seaside_testbed.zip) file).
+5. [MMSA testbed](notebooks/Seaside_testbed/Seaside_testbed.md) ([zip](https://github.com/IN-CORE/incore-docs/blob/main/notebooks/MMSA_testbed.zip) file).
```

Comment:
This links to the Seaside_testbed.md

Reply:
There appears to be some magic with the naming. This needs to match the path to the notebook, but with the "md" instead of ipynb and it looks like it gets automatically generated from the notebook. For example, this should be:

notebooks/MMSA_testbed/MMSA_Seismic_Functionality_Analysis_for_Interdependent_Buildings-Water-Power_using_Fragility_and_Repair_Rate_Curves.md